management/management_option.go

```diff
@@ -93,6 +93,18 @@ func WithAuth0ClientInfo(auth0ClientInfo client.Auth0ClientInfo) Option {
 	}
 }
 
+// WithAuth0ClientEnvOption allows adding extra environment keys to the client information.
+func WithAuth0ClientEnvOption(key string, value string) Option {
```

Comment:
Just curious, why call it "Option"?

Reply:
I wanted to try and convey this was to add to the env object rather than replace it, maybe `WithAuth0ClientEnvEntry` fits better? I wasn't sure on whether `Key` or `Value` fully suited